compiler/rustc_query_impl/src/plumbing.rs

```diff
@@ -466,7 +466,14 @@ macro_rules! define_queries {
         }
 
         impl<'tcx> QueryDescription<QueryCtxt<'tcx>> for queries::$name<'tcx> {
-            rustc_query_description! { $name }
+            fn describe(tcx: QueryCtxt<'tcx>, key: Self::Key) -> String {
```

Comment:
We never call `describe` through the trait. Could you remove it from the trait and make access more direct?